From 38e21b6510971dfb59ae3d18b76aa89bc62b5861 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Fri, 31 Aug 2018 00:20:05 +0100
Subject: [PATCH 1/2] LockClock: Declare private API dependency

 * Addresses the following compilation issue:

   error: LockClock (java:sdk) should not link to org.lineageos.platform.internal (java:platform)

Change-Id: Ic3635deb1f9842aa82e90dd2c8b4a04ceb8f6654
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index ef36048..4b70ffb 100644
--- a/Android.mk
+++ b/Android.mk
@@ -19,7 +19,7 @@ include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES := $(call all-java-files-under, src)
 
-LOCAL_SDK_VERSION := current
+LOCAL_PRIVATE_PLATFORM_APIS := true
 LOCAL_PACKAGE_NAME := LockClock
 LOCAL_PROGUARD_FLAG_FILES := proguard.flags
 
-- 
2.17.1

